### DISCUSSION TITLE
bouton sticky

### DISCUSSION THREAD

**Discussion by Marcos-MEDENOU** [filamentphp/filament#8031] [General]
Comment je peux mettre un formulaire en position sticky sur une ressources ? En faite, je veux ajouter un bouton e bas de page qui au click doit scroller a l'en tète de page peut importe ou on se situe sur la page

**Comment by micraux** [comment]
`French`: Bonjour, c'est un forum international ici, donc merci de rédiger tes messages en **Anglais** :-)

`English`: Please write in English, not in French :-)
